joomla/joomla-cms
# Addition of a new module submenu under administrator extension drop down menu.

#### Summary of Changes

1.Added a new sub menu under module option provided in extension drop down menu.
2.Changes made can also be seen in Hathor template.
#### Testing Instructions

1.Before applying patch no sub menu under module option provided in extension drop down menu.
2.Apply patch
3.New submenu to Add New Module is created.
4.Check with hathor template also.
